Core/src/main/resources/lang/messages_en.json

```diff
@@ -381,6 +381,9 @@
   "info.plot_list_default": "<gold><plot></gold>",
   "info.plot_list_player_online": "<dark_aqua><prefix></dark_aqua><hover:show_text:\"<dark_aqua>Online</dark_aqua>\"><gold><player></gold></hover>",
   "info.plot_list_player_offline": "<dark_aqua><prefix></dark_aqua><hover:show_text:\"<dark_gray>Offline</dark_gray>\"><gold><player></gold></hover>",
+  "info.plot_list_player_unknown": "<hover:show_text:\"<red>The owner of this plot is unknown</red>\"><white><player></white></hover>",
+  "info.plot_list_player_server": "<hover:show_text:\"<red>The plot is owned by the server</red>\"><white><player></white></hover>",
+  "info.plot_list_player_everyone": "<hover:show_text:\"<blue>The plot is owned by everyone</blue>\"><white><player></white></hover>",
```

Comment:
Why does the player placeholder need to be present here? The lack of a space is also not ideal if it does need to be there for whatever reason

Reply:
It is the same scheme of the existing two entries above. It shows the plot list with the plot-owners.

Reply:
The comment is RE all three, and they would result in "The owner of the plot is unknownunknown" for example

Reply:
The player Placeholder is so the Name is consistent with the corresponding names out of the box (`info.unknown`,`info.server` and `info.everyone`).

Reply:
It is still duplicating the "unknown"/"server"/"everyone". You have those provided by the string itself and the placeholder

Reply:
In what way is it duplicating?

Reply:
![image](https://user-images.githubusercontent.com/46501008/166707029-d24206bc-437b-4602-aa9e-c88e7ec1c1f6.png)

Reply:
AH I see it's a hover... Somehow missed that each time...